Add missing 3CB and RHS vehicles cargo logistics nodes

jwoodruff40 opened the pull request:
## What type of PR is this?
1. [ ] Bug
2. [ ] Change
3. [x] Enhancement
4. [x] Miscellaneous

### What have you changed and why?
Information:

Adds missing cargo nodes for:

3CB:

- Datsun (covered)
- Hilux (covered)
- Land Rover (unarmed, open and covered)

RHS:

- M998 4 door
- M1025 (incl armed variants)
- M1043 (incl armed variants)
- M1151 (incl armed variants)
- M1240 (incl armed variants)
- UAZ (open)

### Please specify which Issue this PR Resolves (If Applicable).
https://discord.com/channels/817005365740044289/1218566036459229245/1330269414863409224

### Please verify the following.

1. [x] Have you loaded the mission in LAN host?
2. [ ] Have you loaded the mission on a dedicated server?

### Is further testing or are further changes required?

1. [x] No
2. [ ] Yes (Please provide further detail below.)

### How can the changes be tested?
Steps:

********************************************************
Notes:

jwoodruff40 committed:
Add missing 3CB / RHS vehicles logistics nodes

Silence-112 reviewed (commented):
Looks fine from a code perspective, node placement should be double checked by someone (@stutpip123)?

jwoodruff40 commented:
> Looks fine from a code perspective, node placement should be double checked by someone (@stutpip123)?

I gathered stut wants nothing to do with RHS lol

Also, be aware that some large cargo can clip through the roof of the covered trucks. Perhaps that's why they weren't included in the first place? Fine with little stuff like the loot box.

Shouldn't be an issue with the HMMWV trunks since they should just be one cargo space/slot/whatever we call it so they can't load the big stuff.

UAC-MaxxLite commented:
2 of the following vics dont have storage nodes.
"rhsusf_m1151_m2crows_usarmy"
"rhsusf_m1151_m2_lras3_v1_usarmy"
is that by design?
![image](https://github.com/user-attachments/assets/94e3b54f-cce5-433e-b129-7f13e6601829)
![image](https://github.com/user-attachments/assets/76fda4b5-171d-4c45-a01c-a6c56ae552bd)

jwoodruff40 commented:
> 2 of the following vics dont have storage nodes.
> "rhsusf_m1151_m2crows_usarmy"
> "rhsusf_m1151_m2_lras3_v1_usarmy"
> is that by design?
> ![image](https://github.com/user-attachments/assets/94e3b54f-cce5-433e-b129-7f13e6601829)
> ![image](https://github.com/user-attachments/assets/76fda4b5-171d-4c45-a01c-a6c56ae552bd)
> 
> 

~~I remember there were a couple I skipped because they had ammo stored in the trunk so it didn't make sense to have cargo clip through that. Don't remember if it was those specifically. If not I can add them when I get home.~~ nevermind the second pic just loaded. I'll take care of them later unless you want to. Most of the 1151s can be inherited from the main one but a couple are different

UAC-MaxxLite commented:
on the uaz you will need to move the node more forward as it clips through the back
![image](https://github.com/user-attachments/assets/b056c662-7c8a-41a0-9d1c-60b9620c0b4e)
![image](https://github.com/user-attachments/assets/001ef023-9e64-4718-8716-3782b92b993c)
the clipping is small but some people will get pissy over that

UAC-MaxxLite commented:
other then that some of the nodes are uncentered in the trunk (on the M1025 incl armed variants) but its not something which is an issue in my eyes
![image](https://github.com/user-attachments/assets/f1ab66c3-75ea-4f80-bbd1-b9e16e2c93a8)

UAC-MaxxLite commented:
> > 2 of the following vics dont have storage nodes.
> > "rhsusf_m1151_m2crows_usarmy"
> > "rhsusf_m1151_m2_lras3_v1_usarmy"
> > is that by design?
> > ![image](https://private-user-images.githubusercontent.com/162753200/406687864-94e3b54f-cce5-433e-b129-7f13e6601829.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc4MzkzNDIsIm5iZiI6MTczNzgzOTA0MiwicGF0aCI6Ii8xNjI3NTMyMDAvNDA2Njg3ODY0LTk0ZTNiNTRmLWNjZTUtNDMzZS1iMTI5LTdmMTNlNjYwMTgyOS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEyNVQyMTA0MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT1jZmFkMWI1MjhkYTNjZDFiMzYzNGQ2MmI3ZWMxMGQ4MGQ5NjVlYWQwNDZmMWZjNzAyYTA4YmYxYTA2OGVkZDM2JlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.CXLnOKhREjIOUqbiquyapzX-XvQcIo_fdedSh9HGLho)
> > ![image](https://private-user-images.githubusercontent.com/162753200/406687887-76fda4b5-171d-4c45-a01c-a6c56ae552bd.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3Mzc4MzkzNDIsIm5iZiI6MTczNzgzOTA0MiwicGF0aCI6Ii8xNjI3NTMyMDAvNDA2Njg3ODg3LTc2ZmRhNGI1LTE3MWQtNGM0NS1hMDFjLWE2YzU2YWU1NTJiZC5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMTI1JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDEyNVQyMTA0MDJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT01NmM5ODg0MzUyNWRjYzM0ZGFlNDVjNWI0YTMxM2I2YjVkZTg3NjA3MWVhYzJkZDY0N2NkNGM2MDYyYTgxZWJkJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.bwOmAJpfInZVoHv21OWfFxXWAPGJVAMA5OSAzhK8ykg)
> 
> ~I remember there were a couple I skipped because they had ammo stored in the trunk so it didn't make sense to have cargo clip through that. Don't remember if it was those specifically. If not I can add them when I get home.~ nevermind the second pic just loaded. I'll take care of them later unless you want to. Most of the 1151s can be inherited from the main one but a couple are different

rn doing some testing so cant do it myself lad sorry.
after it if i still have some time then sure

jwoodruff40 committed:
Update RHS vehicle logistics nodes.

- Move most of HMMWVs to left slightly
- Add MATVs
- Add missing HMMWVs (CROWS)
- Move UAZ open forward slightly

jwoodruff40 committed:
Merge branch 'unstable' into 3cb-log-nodes

jwoodruff40 reviewed (commented):
@UAC-MaxxLite Done. o7

- Added the missing 1151s
- Added 1043s (same as 1025s)
- Moved almost all HMMWVs slightly to left to center them
- Moved UAZ open forward slightly to stop clipping
- Added M1240 MATVs
- Merged current unstable

Silence-112 reviewed (approved):
lgtm
